users/models.py

```diff
@@ -61,6 +62,7 @@ class Post(models.Model):
     post = models.TextField()
     author = models.ForeignKey(User, on_delete=models.CASCADE)
     created_at = models.DateTimeField(verbose_name="Date Created", default=timezone.now)
+    # parking_space = models.ForeignKey(ParkingSpace, on_delete=models.CASCADE)
```

Comment:
We should also talk tonight on whether we delete all posts after a user has deleted their account. I am realizing that we may not want to.